lib/m2ts/m2ts.js

```diff
@@ -33,6 +32,9 @@ var
 TransportPacketStream = function() {
   var
     buffer = new Uint8Array(MP2T_PACKET_LENGTH),
+    totalDataEmitted = 0,
+    sinceFlushDataEmitted = 0,
+    invalidPushCalls = 0,
```

Comment:
I think that we do need one of these stats (sinceFlushDataEmitted) so that we can make sure that we emitted data before flush was called. I am fine getting rid of the others. This would mean keeping triggerData_ or overloading trigger, checking if the event is data and incrementing sinceFlushDataEmitted, then calling the prototype function with the same arguments. Unless we don't think that checking to see if we actually emitted data before a flush is an issue.